tensorflow_addons/optimizers/conditional_gradient_test.py

```diff
@@ -14,41 +14,56 @@
 # ==============================================================================
 """Tests for Conditional Gradient."""
 
+import numpy as np
+import pytest
+import platform
+
 import tensorflow as tf
 from tensorflow_addons.utils import test_utils
-import numpy as np
 from tensorflow_addons.optimizers import conditional_gradient as cg_lib
-import platform
 
 
-@test_utils.run_all_in_graph_and_eager_modes
-class ConditionalGradientTest(tf.test.TestCase):
+WINDOWS_CPU = platform.system() == "Windows"
+
+pytestmark = pytest.mark.skipif(
+    WINDOWS_CPU,
+    reason="Doesn't work on Windows CPU version, tf.half is not registered for tf.linalg.svd function on Windows CPU version. So we have to remove tf.half when testing with Windows CPU version. See https://github.com/tensorflow/tensorflow/issues/36764",
+)
+
 
-    # Based on issue #347 in the following link,
-    #        "https://github.com/tensorflow/addons/issues/347"
-    # tf.half is not registered for 'ResourceScatterUpdate' OpKernel
-    # for 'GPU' devices.
-    # So we have to remove tf.half when testing with gpu.
-    # The function "_DtypesToTest" is from
-    #       "https://github.com/tensorflow/tensorflow/blob/5d4a6cee737a1dc6c20172a1dc1
-    #        5df10def2df72/tensorflow/python/kernel_tests/conv_ops_3d_test.py#L53-L62"
-    def _DtypesToTest(self, use_gpu):
-        if use_gpu:
-            return [tf.float32, tf.float64]
-        else:
-            return [tf.half, tf.float32, tf.float64]
-
-    # Based on issue #36764 in the following link,
-    #        "https://github.com/tensorflow/tensorflow/issues/36764"
-    # tf.half is not registered for tf.linalg.svd function on Windows
-    # CPU version.
-    # So we have to remove tf.half when testing with Windows CPU version.
```

Comment:
Can you also keep this comment in the new version? It's handy to understand the code.